### ISSUE TITLE
&lt; &gt; and &amp; character codes error.

### ISSUE THREAD

**Issue by hakantr** [mdn/content#591]
content/files/en-us/web/css/css_background_and_borders/box-shadow_generator/index.html => `<pre class = "brush: js"> <code class = "language-js"> ....
this.value = value &lt;= max &amp;&amp; value &gt;= min ? value : (min + max) / 2 | 0;
.... </code> </pre>`

"& lt; & gt; and & amp;" in javascript codecs custom definitions give "error: Invalid regular expression".

**Comment by sideshowbarker** [comment]
Thanks for reporting this. I’ve opened https://github.com/mdn/content/pull/611 with a fix.

**Comment by hakantr** [comment]
Hello there;
I gave that single line as an example. In fact, there are more of these character references in the code. Below is a list of what I found.
Good luck with.
```
/**
 * UI-SlidersManager
 */

var SliderManager = (function SliderManager() {
.
.
.
		this.max = max &gt; 0 ? max : 100;
        .
        .
        .
		if (slider.value + slider.step &lt;= slider.max) {
            .
            .
            .
		if (slider.value - slider.step &gt;= slider.min) {
            .
            .
            .
		if (pos &lt; 0) pos = 0;
		if (pos &gt; width) pos = width;
        .
        .
        .
		if (precision &gt; this.step / 2)
        .
        .
        .
		if (value &gt; slider.max || value &lt; slider.min)
        .
        .
        .
		for (var i = 0; i &lt; size; i++)
        .
        .
        .
		for (var i = 0; i &lt; size; i++)
        .
        .
        .
		for (var i = 0; i &lt; size; i++)
        .
        .
        .
		for (var i = 0; i &lt; subscribers[this.topic].length; i++)
        .
        .
		for (var i = 0; i &lt; size; i++)
        .
        .
		if (H &gt;= 0 &amp;&amp; H &lt; 1) {	this.setRGBA(C, X, m);	return; }
		if (H &gt;= 1 &amp;&amp; H &lt; 2) {	this.setRGBA(X, C, m);	return; }
		if (H &gt;= 2 &amp;&amp; H &lt; 3) {	this.setRGBA(m, C, X);	return; }
		if (H &gt;= 3 &amp;&amp; H &lt; 4) {	this.setRGBA(m, X, C);	return; }
		if (H &gt;= 4 &amp;&amp; H &lt; 5) {	this.setRGBA(X, m, C);	return; }
		if (H &gt;= 5 &amp;&amp; H &lt; 6) {	this.setRGBA(C, m, X);	return; }
        .
        .
		if (this.hue &lt; 0) this.hue += 360;
        .
        .
		if (this.r &lt; 16) r = '0' + r;
		if (this.g &lt; 16) g = '0' + g;
		if (this.b &lt; 16) b = '0' + b;
        .
        .
			if (x &gt; size)
            .
			if (y &gt; size)
            .
			if (x &lt; 0) x = 0;
			if (y &lt; 0) y = 0;
            .
            .
			if (x &lt; 0) x = 0;
			if (x &gt; width) x = width;
            .
            .
			if (x &lt; 0) x = 0;
			if (x &gt; width) x = width;
            .
            .
			if (typeof value === 'number' &amp;&amp; isNaN(value) === false &amp;&amp;
				value &gt;= 0 &amp;&amp; value &lt; 360)
                .
                .
			if (typeof value === 'number' &amp;&amp; isNaN(value) === false &amp;&amp;
				value &gt;= 0 &amp;&amp; value &lt;= 255)
                .
                .
			if (typeof value === 'number' &amp;&amp; isNaN(value) === false &amp;&amp;
				value &gt;= 0 &amp;&amp; value &lt;= 1)
                .
                .
			if (this.layer === null &amp;&amp; this.stack.children.length &gt;= 1)
            .
            .
			if (this.count &lt;= 1 || this.layer === null)
				return;
			if (direction === -1 &amp;&amp; this.layerID === (this.count - 1) )
				return;
			if (direction === 1 &amp;&amp; this.layerID === 0 )
            .
            .
			if (index &gt; this.layerID)
				return;
                .
				if (this.count &gt;= 1) {
                .
			if (index &lt; this.layerID) {
            .
            .
			for (var i = 0; i &lt; size; i++) {
                .
                .
				if (value === false &amp;&amp; active.stack === stacks['before'])
					setActiveStack(stacks['element'])
				if (value === true &amp;&amp; active.stack !== stacks['before'])
                .
                .
				if (value === false &amp;&amp; active.stack === stacks['after'])
					setActiveStack(stacks['element'])
				if (value === true &amp;&amp; active.stack !== stacks['after'])
                .
                .
			for (var i = 0; i &lt; size; i++).
            .
            .
            .
            
</code></pre>
```

**Comment by sideshowbarker** [comment]
> I gave that single line as an example. In fact, there are more of these character references in the code.

Thanks, yeah I noticed that too — but only after writing up this PR. So I’ll do a follow-up PR to fix the rest.
